src/test/java/com/tom/test/excercise01/ShoppingCartTest.java

```diff
@@ -0,0 +1,55 @@
+package com.tom.test.excercise01;
+
+import com.tom.driver.manager.DriverUtils;
+import com.tom.page.object.Excercise01Page;
+import com.tom.test.TestBase;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShoppingCartTest extends TestBase {
```

Comment:
junit5 does not require class to be public. Same goes for methods. Just remove public from class and methods